src/freenet/client/filter/CSSTokenizerFilter.java

```diff
@@ -1722,18 +1722,18 @@ public static String HTMLelementVerifier(String elementString, boolean isIDSelec
 		ArrayList<String> attSelections = null;
 		while(elementString.indexOf('[')!=-1 && elementString.indexOf(']')!=-1 && (elementString.indexOf('[')<elementString.indexOf(']')))
 		{
-		    if(isIDSelector) return null;
+			if(isIDSelector) return null;
 			String attSelection=elementString.substring(elementString.indexOf('[')+1,elementString.indexOf(']')).trim();
 			StringBuilder buf=new StringBuilder(elementString);
 			buf.delete(elementString.indexOf('['), elementString.indexOf(']')+1);
 			elementString=buf.toString();
-			if(logDEBUG) Logger.debug(CSSTokenizerFilter.class, "attSelection="+attSelection+"  elementString="+elementString);
+			if(logDEBUG) Logger.debug(CSSTokenizerFilter.class, "attSelection="+attSelection+"	elementString="+elementString);
```

Comment:
I think you may have been a bit overzealous with the tabifying, usually only multiple _leading_ spaces are turned into tabs… 😄

Reply:
should be fixed now — thank you for your mind for the detail!